.gitignore

```diff
@@ -1,3 +1,4 @@
 data/
 network/
 .python-version
+k8s/.env
```

Comment:
Should we add a `.env-example` to help with discoverability of our secrets configuration?